pkg/storagev2/inmemstore/inmembatch.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2022 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package inmemstore
+
+import (
+	"context"
+	"errors"
+	"sync"
+
+	storage "github.com/ethersphere/bee/pkg/storagev2"
+)
+
+type batchOp struct {
+	delete bool
+	item   storage.Item
+}
+
+type Batch struct {
+	mu    sync.Mutex
+	ctx   context.Context
+	ops   map[string]batchOp
+	store *Store
+	done  bool
+}
+
+func (s *Store) Batch(ctx context.Context) (storage.Batch, error) {
+	return &Batch{
+		ctx:   ctx,
+		ops:   make(map[string]batchOp),
+		store: s,
+	}, nil
+}
+
+func (i *Batch) Put(item storage.Item) error {
+	if i.ctx.Err() != nil {
+		return i.ctx.Err()
+	}
+
+	i.mu.Lock()
+	defer i.mu.Unlock()
+
+	i.ops[item.ID()] = batchOp{
```

Comment:
For delete operation, key in `ops` map is generated using ` getKeyString(key)` method. Here in put operation, key in `ops` map is just `item.ID()` value. Is there reason why ` getKeyString(key)` is not used here?